ios/FluentUI/Vnext/Button/Button.swift

```diff
@@ -3,21 +3,101 @@
 //  Licensed under the MIT License.
 //
 
-import Combine
-import UIKit
 import SwiftUI
+import UIKit
+
+/// View that represents the button
+public struct MSFButtonView: View {
+    @Environment(\.theme) var theme: FluentUIStyle
+    @Environment(\.windowProvider) var windowProvider: FluentUIWindowProvider?
+    @ObservedObject var tokens: MSFButtonTokens
+    @ObservedObject var state: MSFButtonStateImpl
+    var action: () -> Void
+
+    public init(action: @escaping () -> Void,
```

Comment:
Might be nice to have the action be the final parameter to the initializer so we can create the control with a trailing closure:
```swift
MSFButton(text: "Title") {
    // action stuff!
}
```